src/index.html

```diff
@@ -7,6 +7,489 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <img src="./images/logo.png" alt="logo">
+      </div>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item" data-qa="hover">
```

Comment:
![image](https://user-images.githubusercontent.com/24947782/188964123-003eb542-386d-42f6-9d13-0b1a869630a8.png)

Reply:
fix it